lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -18,21 +18,20 @@
 def _get_testable_interactive_backends():
     backends = []
     # gtk3agg fails on Travis, needs to be investigated.
```

Comment:
Move this comment next to the commented out line in the for list. Otherwise it might be overseen and it becomes unclear why gtk3agg is commented out in the list.

Reply:
sure...